templates/spec/helpers/taxons_helper_spec.rb

```diff
@@ -4,11 +4,10 @@
 
 RSpec.describe TaxonsHelper, type: :helper do
   describe '#taxon_seo_url' do
-    let(:taxon_permalink) { 'ruby-on-rails' }
-    let(:taxon) { create(:taxon, permalink: taxon_permalink) }
+    let(:taxon) { create(:taxon) }
 
     it 'is the nested taxons path for the taxon' do
-      expect(taxon_seo_url(taxon)).to eq("/t/#{taxon_permalink}")
+      expect(taxon_seo_url(taxon)).to eq("/t/#{taxon.permalink}")
```

Comment:
Is this just a refactoring or there was something wrong with the original version?

I'm concerned that if the permalink is blank it will pass, but for the wrong reason

Reply:
I agree 👍 I also had doubts about this part.
The previous method of assigning a permalink to the taxon did not accurately reflect the current logic for generating taxon permalinks, which always includes a taxonomy now:
As a result, the default URL structure was incorrect and should have been something like `"brand-1/ruby-on-rails"` instead of just `"ruby-on-rails"`.

I have recreated the taxonomy and taxon using more appropriate names.
Please let me know if you have any feedback (or if we prefer to simply verify that the `taxon_seo_url` function returns the same value assigned to `taxon.permalink).